src/wireclient.js

```diff
@@ -93,10 +93,19 @@
   }
 
   function readBinaryData(content, mimeType, callback) {
-    var blob = new Blob([content], { type: mimeType });
+    var blob;
+    global.BlobBuilder = global.BlobBuilder || global.WebKitBlobBuilder;
```

Comment:
Does the resulting var need to be global? This *might* cause issues with other libs, no?